tests/org.jboss.tools.jsf.ui.test/src/org/jboss/tools/jsf/ui/test/project/RunJSFProjectTest.java

```diff
@@ -50,9 +57,13 @@
 @DoNotUseVPE
 @UseParametersRunnerFactory(ParameterizedRequirementsRunnerFactory.class)
 @JBossServer(state = ServerRequirementState.RUNNING)
+@CloseAllEditors
 public class RunJSFProjectTest {
 
 	protected static final String PROJECT_NAME_BASE = "JSFTestProject";
+	
+	@SuppressWarnings("unused")
+	private static boolean IS_WINDOWS = System.getProperty("os.name").startsWith("Windows") ? true : false;
```

Comment:
Where do you use this?

Reply:
I do not use it anymore, that is why is suppress warning annotation there... will remove.